pkg/source/openstack/client.go

```diff
@@ -355,6 +364,9 @@ func (c *Client) GenerateVirtualMachine(vm *migration.VirtualMachineImport) (*ku
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      vm.Spec.VirtualMachineName,
 			Namespace: vm.Namespace,
+			Annotations: map[string]string{
```

Comment:
we should make this optional for when an actual description is returned, else we just end up with a key but no value.

rest looks good.

Reply:
Done